Client/Frontend/Browser/MainMenuActionHelper.swift

```diff
@@ -270,18 +261,6 @@ class MainMenuActionHelper: PhotonActionSheetProtocol, FeatureFlaggable, CanRemo
 
     // MARK: - Actions
 
-    private func getInviteFriendsAction() -> PhotonRowActions {
-        SingleActionViewModel(
-            title: .localized(.inviteFriends),
-            iconString: "inviteFriends",
-            isNew: User.shared.referralsEntryPointIsNew) { [weak self] action in
-                action.isNew = false
-                self?.delegate?.showReferrals()
-                User.shared.referralsEntryPointUsed()
```

Comment:
If I looked at it right, I think there is another instance of `User.shared.referralsEntryPointUsed()` inside `MultiplyImpact` that is not covered by this revert 🤔 

Would suggest maybe updating core in here after https://github.com/ecosia/ios-core/pull/102 is merged to be sure any potential issue is covered.

Reply:
Well spotted 🙏 Consider it fixed 😌